packages/@guidesmiths/cuckoojs-schematics/src/pr-template/schema.json

```diff
@@ -1,9 +1,15 @@
 {
   "$schema": "http://json-schema.org/schema",
-  "$id": "SchematicsPrTemplate",
+  "$id": "SchematicsCI",
   "title": "PR Template Options Schema",
   "type": "object",
   "properties": {
+    "directory": {
+      "type": "string",
+      "description": "quickstart destination directory",
+      "default": ".",
+      "x-prompt": "Where do you want to create the project?"
+    },
```

Comment:
Is this correct? This schema is for the PR template.

Reply:
I've changed name, 
and about PR template directory yes, it's needed because when we apply with cli the schematics, we do in another folder (the project name), not in the root (it's more a requirement inherited from cli)

Reply:
Understood!